### PR TITLE
retry file copies during migration

### PR DESCRIPTION
**Pull request by dr-bonez**

### CODE REVIEW

File: backend/src/backup/backup_bulk.rs [hunk 1]
```diff
@@ -126,7 +125,7 @@ pub async fn backup_all(
         .load(&mut ctx.secret_store.acquire().await?)
         .await?;
     let mut backup_guard = BackupMountGuard::mount(
-        TmpMountGuard::mount(&fs).await?,
+        TmpMountGuard::mount(&fs, false).await?,
```
Comment (Blu-J) [inline backend/src/backup/backup_bulk.rs:128]: Please make this not a boolean

File: backend/src/disk/mount/filesystem/block_dev.rs [hunk 2]
```diff
@@ -14,13 +14,15 @@ use crate::{Error, ResultExt};
 pub async fn mount(
     logicalname: impl AsRef<Path>,
     mountpoint: impl AsRef<Path>,
+    readonly: bool,
```
Comment (Blu-J) [inline backend/src/disk/mount/filesystem/block_dev.rs:17]: This is the source of evil.

File: backend/src/disk/mount/filesystem/cifs.rs [hunk 3]
```diff
@@ -40,19 +40,22 @@ pub async fn mount_cifs(
     username: &str,
     password: Option<&str>,
     mountpoint: impl AsRef<Path>,
+    readonly: bool,
```
Comment (Blu-J) [inline backend/src/disk/mount/filesystem/cifs.rs:43]: To non bool

**Review by Blu-J — APPROVED** [review]

**Comment by kn0wmad** [comment]
All concerns have been addressed based on my test.  Will test again after merge

**Review by kn0wmad — APPROVED** [review]
All concerns have been addressed based on my test. Will test again after merge

**Pull request merged**